static/src/javascripts/projects/common/modules/commercial/third-party-tags/outbrain.js

```diff
@@ -120,6 +126,30 @@ define([
             detect.adblockInUseSync();
     }
 
+    function checkEmailSignup() {
```

Comment:
I feel like a small improvement would have this function return a new promise on the first call and a cached promised on the second call.